python_modules/dagster-airflow/dagster_airflow/dagster_plugin.py

```diff
@@ -300,15 +451,14 @@ def get_host_tmp_dir(self):
     def execute(self, context):
         if 'dag_run' in context and context['dag_run'] is not None:
             self._run_id = context['dag_run'].run_id
-        else:
-            self._run_id = str(uuid.uuid4())
         try:
-            # FIXME implement intermediate result persistence
+            # FIXME implement intermediate result persistence to S3
             return super(DagsterOperator, self).execute(context)
         finally:
             self._run_id = None
 
     def __get_tls_config(self):
+        # pylint:disable=no-member
```

Comment:
can you comment as to why this is necessary

Reply:
yep